docs/kubernetes-deployment/signer/values/staging/values-signer0-staging.yaml

```diff
@@ -46,8 +46,8 @@ env:
 fullnameOverride: ""
 image:
   pullPolicy: Always
-  repository: us-west1-docker.pkg.dev/devopsre/social-connect/odis-signer
-  tag: "odis-signer-3.1.1"
+  repository: us-west1-docker.pkg.dev/devopsre/dev-images/odis-signer
+  tag: latest
```

Comment:
I'm a bit reluctant to change this tag to `latest`, as the latest image may not always be the one we use in production.

Could you elaborate on why we would want to use the `latest` tag in the signer deployment yaml?

Reply:
One consideration here is that we're using [policy-controller](https://github.com/sigstore/policy-controller) in all our k8s cluster. One of the things that policy-controller does is, whenever you deploy/upgrade anything (in this case the helm chart for odis releases), it gets the `image` field composed by `<repository>:<tag>`, resolves which image sha corresponds, in that moment, to that tag, and places in k8s the image as `<repository>:<sha>`. That would impact that the deployment would only be upgraded to the `latest` image whenever a `helm upgrade` is done on staging (so it should be always a "controlled" process).